addons/mail/static/src/components/chatter/chatter_tests.js

```diff
@@ -452,6 +461,160 @@ QUnit.test('do not post message with "Enter" keyboard shortcut', async function
     );
 });
 
+QUnit.test("Display suggested recipients list", async function (assert) {
```

Comment:
Name is a bit too generic. I would expect test with just a single recipient for such named test. Whether it's with or without partner, it's unclear so another reason name of test need improvement.